Firmware/LowLevel/src/main.cpp

```diff
@@ -56,11 +56,15 @@ SerialPIO uiSerial(PIN_UI_TX, PIN_UI_RX, 250);
 /**
  * @brief Some hardware parameters
  */
-#define VIN_R1 10000.0f
-#define VIN_R2 1000.0f
+#define VIN_R1 9980.0f
+#define VIN_R2 998.0f
```

Comment:
Where do these come from? I assume these will be slightly different for every board (10k 1%, 1k 1%), so we should have them at 10k and 1k